src/NuGet.Core/NuGet.ProjectModel/ProjectCentralTransitiveDependencyGroup.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NuGet.Frameworks;
+using NuGet.LibraryModel;
+using NuGet.Shared;
+
+namespace NuGet.ProjectModel
+{
+    public class ProjectCentralTransitiveDependencyGroup : IEquatable<ProjectCentralTransitiveDependencyGroup>
```

Comment:
Project in the name is redundant....everything is project. 

CentrallyDefinedTransitiveDependencyGroup?

Reply:
Renamed